web-app-router/api/auth/[auth0]/index.ts

```diff
@@ -0,0 +1,39 @@
+import { loginCallback } from "@/api/login-callback";
+
+import {
+  AppRouteHandlerFnContext,
+  HandlerError,
+  handleAuth,
+  handleCallback,
+  handleLogin,
+  handleLogout,
+} from "@auth0/nextjs-auth0";
+
+import { NextRequest, NextResponse } from "next/server";
+
+export const auth0 = handleAuth({
+  login: (req: NextRequest, ctx: AppRouteHandlerFnContext) => {
+    const invite_id = req.nextUrl.searchParams.get("invite_id");
+
+    return handleLogin(req, ctx, {
+      returnTo: invite_id
+        ? `/api/auth/login-callback?invite_id=${invite_id}`
+        : "/api/auth/login-callback",
+    });
+  },
+
+  callback: handleCallback,
+  "login-callback": loginCallback,
+  // TODO: Add delete account handler
+
+  logout: handleLogout({
+    returnTo: "/login",
```

Comment:
are we sure we don't need login_error handling here now?

Reply:
What login error should we have and how it should be displayed?